racket/src/rktio/rktio.h

```diff
@@ -76,11 +76,12 @@ Thread and signal conventions:
 
  - On systems where signal handling is thread-specific, as on Linux,
    then `rktio_init` should be called before any additional threads,
-   so that a suitable inheritable siganl disposition can be
+   so that a suitable inheritable signal disposition can be
    configured.
 
 */
      
+#include <time.h>
```

Comment:
I don't think this needs to be here. A client who uses `"rktio.h"` doesn't need to know about `<time.h>`, right?

Reply:
It seems I added this in [286e2a3b] when I added the timestamp fields to the C and Racket structs. When I check the manpage `man 2 stat`, it doesn't list `time.h` as requirement. The header is only used in some example code of the manpage.

Since everything still works after removing the `time.h` include, I removed it in [286e2a3b].

Reply:
I haven't pushed the review changes yet, so the commit id links will be dead. I'll remove this comment after pushing my changes.